packages/gitgraph-node/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import { GitgraphCore } from "gitgraph-core/lib";
+
+// Domain (rendering logic)
+import computeGraphMap from "./compute-graph-map";
+
+// Infrastructure (logger implementations)
+import consoleGraphLogger from "./console-graph-logger";
+
+const gitgraph = new GitgraphCore();
+
+function render() {
+  const graphMap = computeGraphMap(gitgraph);
+  consoleGraphLogger(graphMap);
+}
+
+export { gitgraph, render };
+export * from "gitgraph-core/lib";
```

Comment:
Because all template values are not relevant here, we can maybe limit the gitgraph options ;)